config/github/codeOwners

```diff
@@ -0,0 +1 @@
+* @mangs @jduthon
```

Comment:
Interesting 🤔 
Looks like I am missing some right I guess, I'll have a look 😅 

I am thinking, ideally we would put our whole team here, so if one of us isn't around things aren't getting blocked?

Reply:
OK. I matched the ESLint list, but happy to add to it.

Reply:
Yea I noticed the error about your account which confused me but I didn't know how to resolve it. Any ideas?

Reply:
I think I noticed last time I am not part of a team for the Babbel org, but didn't take the time to fix it yet 🙈

Reply:
you're part of the [Pineapples team](https://github.com/orgs/lessonnine/teams/pineapples)